src/Mvc/Mvc.TagHelpers/src/LinkTagHelper.cs

```diff
@@ -362,7 +372,9 @@ private void BuildFallbackBlock(TagHelperAttributeList attributes, TagHelperCont
             builder
                 .AppendHtml("<meta name=\"x-stylesheet-fallback-test\" content=\"\" class=\"")
                 .Append(FallbackTestClass)
-                .AppendHtml("\" />");
+                .AppendHtml("\" ")
+                .Append(FallbackTestHtmlAttribute)
```

Comment:
Add an `if (string.IsNullOrEmpty(FallbackTestHtmlAttribute))` for this bit so we can maintain the ending space `_/>`.